GOVERNANCE.md

```diff
@@ -0,0 +1,21 @@
+# Reference Types Working Group Governance
+
+## Discussion
+
+Discussions take place on GitHub's [discussion board](https://github.com/opencontainers/wg-reference-types/discussions) and during [weekly meetings](https://github.com/opencontainers/wg-reference-types/blob/main/README.md#meetings). Notes are taken down at the meetings are recorded in [this google document](https://docs.google.com/document/d/1SVOWQTowigXzbYdorzfa7tMmrcm91yK12LvSONqziJY/edit). Initial seeding of the GitHub discussion topics take place in [this google document](https://docs.google.com/document/d/1ky0LMAvWL_XRnfId9h5GBswV7GGko5GejuyaMWCGX-Y/edit#). It is expected that these google documents will be deprecated in favor of the GitHub Discussion board and GitHub issues.
+
+Discussions shall follow the [neutral framework](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md) previously decided by this working group. Changes to the framework are encouraged via GitHub discussions, issues, and pull requests.
+
+Working group members and maintainers shall follow the [OCI code of conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md)
+
+## Documentation
+
+Meeting minutes shall be published on this GitHub repository. A notetaker shall be identified to take notes during the meetings. Consensus shall be documented in the form of GitHub issues that can be referenced in pull requests.
```

Comment:
Where and how should this happen? Is there are tool that converts Gdocs to mardown that we can use to streamline this process?

Reply:
reg. where and how: Do you have any suggestions?
reg. tools: I don't know of any tools. I have previously used hackmd and etherpad to take notes. Perhaps we can migrate our note taking to one of these tools.